connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectService.scala

```diff
@@ -259,13 +281,15 @@ object SparkConnectService {
     if (debugMode) {
       sb.addService(ProtoReflectionService.newInstance())
     }
+
     server = sb.build
     server.start()
+    createListenerAndUI(sc)
```

Comment:
nit: maybe move `createListenerAndUI` to `start`, just like `detach` is in `stop`. Then don't need to pass sc to `startGRPCService`

Reply:
Updated.